src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,2389 @@
+			UnitSensibleCoolingEnergy(0.0),
+			UnitLatentCoolingRate(0.0),
+			UnitLatentCoolingEnergy(0.0),
+			SystemTotalCoolingRate(0.0),
+			SystemTotalCoolingEnergy(0.0),
+			SystemSensibleCoolingRate(0.0),
+			SystemSensibleCoolingEnergy(0.0),
+			SystemLatentCoolingRate(0.0),
+			SystemLatentCoolingEnergy(0.0),
+			UnitTotalHeatingRate(0.0),
+			UnitTotalHeatingEnergy(0.0),
+			UnitSensibleHeatingRate(0.0),
+			UnitSensibleHeatingEnergy(0.0),
+			UnitLatentHeatingRate(0.0),
+			UnitLatentHeatingEnergy(0.0),
+			SystemTotalHeatingRate(0.0),
+			SystemTotalHeatingEnergy(0.0),
+			SystemSensibleHeatingRate(0.0),
+			SystemSensibleHeatingEnergy(0.0),
+			SystemLatentHeatingRate(0.0),
+			SystemLatentHeatingEnergy(0.0),
+			RequestedLoadToHeatingSetpoint(0.0),
+			RequestedLoadToCoolingSetpoint(0.0),
+			SupplyFanElectricPower(0.0),
+			SupplyFanElectricEnergy(0.0),
+			SecondaryFuelConsumptionRate(0.0),
+			SecondaryFuelConsumption(0.0),
+			ThirdFuelConsumptionRate(0.0),
+			ThirdFuelConsumption(0.0),
+			WaterConsumptionRate(0.0),
+			WaterConsumption(0.0),
+			ExternalStaticPressure(0.0),
+			RequestedHumdificationMass(0.0),
+			RequestedHumdificationLoad(0.0),
+			RequestedHumdificationEnergy(0.0),
+			RequestedDeHumdificationMass(0.0),
+			RequestedDeHumdificationLoad(0.0),
+			RequestedDeHumdificationEnergy(0.0),
+			QLatentZoneOut( 0),
+			QSensZoneOut(0),
+			TsaMin_schedule_pointer(0),
+			TsaMax_schedule_pointer(0),
+			RHsaMin_schedule_pointer(0),
+			RHsaMax_schedule_pointer(0),
+			PrimaryMode(0),
+			PrimaryModeRuntimeFraction(0.0),
+			averageOSAF(0),
+			ErrorCode(0),
+			InletNode(0),
+			OutletNode(0),
+			SecondaryInletNode(0),
+			SecondaryOutletNode(0),
+			FinalElectricalPower(0.0),
+			FinalElectricalEnergy(0.0),
+			InletMassFlowRate(0.0),
+			InletTemp(0.0),
+			InletWetBulbTemp(0.0),
+			InletHumRat(0.0),
+			InletEnthalpy(0.0),
+			InletPressure(0.0),
+			InletRH(0.0),
+			OutletMassFlowRate(0.0),
+			OutletVolumetricFlowRate(0.0),
+			OutletTemp(0.0),
+			OutletWetBulbTemp(0.0),
+			OutletHumRat(0.0),
+			OutletEnthalpy(0.0),
+			OutletPressure(0.0),
+			OutletRH(0.0),
+			SecInletMassFlowRate(0.0),
+			SecInletTemp(0.0),
+			SecInletWetBulbTemp(0.0),
+			SecInletHumRat(0.0),
+			SecInletEnthalpy(0.0),
+			SecInletPressure(0.0),
+			SecInletRH(0.0),
+			SecOutletMassFlowRate(0.0),
+			SecOutletTemp(0.0),
+			SecOutletWetBulbTemp(0.0),
+			SecOutletHumRat(0.0),
+			SecOutletEnthalpy(0.0),
+			SecOutletPressure(0.0),
+			SecOutletRH(0.0),
+			Wsa(0.0),
+			SupplyVentilationAir(0.0),
+			SupplyVentilationVolume(0.0),
+			OutdoorAir(false),
+			MinOA_Msa(0.0),
+			OARequirementsPtr(0),
+			Tsa(0.0),
+			ModeCounter(0),
+			CoolingRequested(false),
+			HeatingRequested(false),
+			VentilationRequested(false),
+			DehumidificationRequested(false),
+			HumidificationRequested(false),
+			//debug
+			RawHR(0)
+		{
+			WarnOnceFlag = false;
+			cp = 1; //kJ/degreesC.kg default value, reset during calculation
```

Comment:
Specific heat in E+ is commonly held as J/kgK, not kJ.  Unless there is a valid reason to keep it as kJ, it needs to change to J.

Reply:
Fixed, all Cp's now in J/kgK